src/client/index.js

```diff
@@ -1 +1,28 @@
 console.log('this is working!!')
+window.submitUsername=function(){
```

Comment:
dude :-1: , why are u manually appending to window object? It just gives global scope anyways.